### DISCUSSION TITLE
Primo approccio alla domotica

### DISCUSSION THREAD

**Discussion by 2Max2Pax** [indomus/forum#520] [GENERALE - Discussioni generiche]
Ciao a tutti, sto iniziando il progetto di domotica per casa. Avrei bisogno di qualche consiglio.
Inizierei con Smart Life e Alexa ma poi vorrei passare a HA o altro sistema in locale.
Il sistema dovrebbe comprendere anche alcuni componenti sui quali chiedo consiglio:
- Alcuni **switch VIMAR 14592** (zigbee)
Sono noti problemi di integrazione con HA o altri Hub personali?
-**Attuatori IR x condizionatori**
Posso fidarmi ad acquistare roba economica da AliExpress?
- **Controllo tende da sole Somfy** (non ancora installate)
Qui ho il dubbio più grande perché secondo l'installatore sono possibili due soluzioni: comandare con radiocomando oppure con pulsanti a muro. Sembra che per passare da una soluzione all'altra si debbano cambiare i motori ed i sensori vento.

Scusate se mi sono dilungato.
Massimiliano

**Comment by zamby** [comment]
Ciao,
anch'io sto valutando i prodotti vimar bt / zigbee e vorrei anch'io l'integrazione con HA.
Sta iniziando un po' maluccio, l'attuatore 19593 sembra non venir riconosciuto dal gateway tuya.
Quindi sono alla ricerca di una strada che mi porti a farli funzionare in altro modo.
Se non ho capito male, leggendo qua  e là, una strada possibile, sembra:
 Interruttore 19593 -> Gateway Vimar BT -> HomeKit -> HA 
oppure
 Interruttore 19593 -> conbee2 -> zigbee2mqtt ->HA

Mi chiedo se qualcuno può confermare le soluzioni, sei riuscito a integrare il deviatore 14592 ad HA ?
Grazie
Andrea

**Comment by sky7176** [comment]
Ciao,
io ho integrato il misuratore di consumi Legrand 412015 e l'ho fatto con la scheda usb CC2531 e zigbee2mqtt.
In questo modo non ho hub ulteriori da gestire.

Con questa integrazione gestisco anche altri dispositivi zigbee, come quelli Ikea o della LIDL e non ho grosse difficoltà, l'unico limite di questa chiavetta è il numero dei dispositivi gestibile che è pari a 20.

In merito al Legrand 412015 non mi ha soddisfatto, visto il suo costo pensavo meglio.

Sono un principiante pure io ma se vi serve una mano chiedete pure.

Giuseppe
